apps/userdashboard/assets/js/a4_candy_userdashboard/ModerationComment.jsx

```diff
@@ -42,12 +44,31 @@ export default class ModerationComment extends Component {
             {userImageDiv}
           </div>
           <div className="col-7 col-md-8">
-            <div><span className="fa-stack fa-2x" aria-hidden="true"><i className="fas fa-exclamation fa-stack-1x" /><i className="far fa-circle fa-stack-2x" /></span>
+            <div><i className="fas fa-exclamation-circle me-1" aria-hidden="true" />
               {userProfileUrl ? <a href={userProfileUrl}>{userName}</a> : userName}
               {aiClassified ? this.getLink(offensiveTextAIInterpolated, commentUrl) : this.getLink(offensiveTextReportInterpolated, commentUrl)}
             </div>
             <div>{created}</div>
           </div>
+          <div className="col">
+            <div className="text-end">
+              <button
+                type="button" className="dropdown-toggle btn btn--link" aria-haspopup="true"
+                aria-expanded="false" data-bs-toggle="dropdown"
+              >
+                <i className="fas fa-ellipsis-v" aria-hidden="true" />
+              </button>
+              <ul className="dropdown-menu dropdown-menu-end">
+                <li key="1">
+                  <button className="dropdown-item" type="button">{archiveText}</button>
```

Comment:
@Rineee here is where you can add the archive action then I can take over again if you like and set correct buttons to show and when to enable/disable them, currently everything is shown